Index: newblog/assets/src/bootstrap/less/bootstrap.less
===================================================================
--- newblog.orig/assets/src/bootstrap/less/bootstrap.less	2013-01-08 21:17:52.000000000 +0100
+++ newblog/assets/src/bootstrap/less/bootstrap.less	2013-01-08 21:18:32.000000000 +0100
@@ -27,7 +27,7 @@
 @import "tables.less";
 
 // Components: common
-@import "sprites.less";
+@import "font-awesome.less";
 @import "dropdowns.less";
 @import "wells.less";
 @import "component-animations.less";
Index: newblog/assets/src/bootstrap/Makefile
===================================================================
--- newblog.orig/assets/src/bootstrap/Makefile	2013-01-08 21:17:52.000000000 +0100
+++ newblog/assets/src/bootstrap/Makefile	2013-01-08 21:18:32.000000000 +0100
@@ -22,7 +22,7 @@
 	@recess --compile ${BOOTSTRAP_RESPONSIVE_LESS} > ${BOOTSTRAP_RESPONSIVE}
 	@echo "Compiling LESS with Recess...               ${CHECK} Done"
 	@node docs/build
-	@cp img/* docs/assets/img/
+	@cp font/* docs/assets/font/
 	@cp js/*.js docs/assets/js/
 	@cp js/tests/vendor/jquery.js docs/assets/js/
 	@echo "Compiling documentation...                  ${CHECK} Done"
@@ -63,10 +63,10 @@
 #
 
 bootstrap:
-	mkdir -p bootstrap/img
+	mkdir -p bootstrap/font
 	mkdir -p bootstrap/css
 	mkdir -p bootstrap/js
-	cp img/* bootstrap/img/
+	cp font/* bootstrap/font/
 	recess --compile ${BOOTSTRAP_LESS} > bootstrap/css/bootstrap.css
 	recess --compress ${BOOTSTRAP_LESS} > bootstrap/css/bootstrap.min.css
 	recess --compile ${BOOTSTRAP_RESPONSIVE_LESS} > bootstrap/css/bootstrap-responsive.css
