Index: newblog/assets/src/bootstrap/less/bootstrap.less
===================================================================
--- newblog.orig/assets/src/bootstrap/less/bootstrap.less	2013-05-11 17:08:57.000000000 +0200
+++ newblog/assets/src/bootstrap/less/bootstrap.less	2013-05-11 17:12:41.000000000 +0200
@@ -27,7 +27,7 @@
 @import "tables.less";
 
 // Components: common
-@import "sprites.less";
+@import "font-awesome.less";
 @import "dropdowns.less";
 @import "wells.less";
 @import "component-animations.less";
Index: newblog/assets/src/bootstrap/Makefile
===================================================================
--- newblog.orig/assets/src/bootstrap/Makefile	2013-05-11 17:08:56.000000000 +0200
+++ newblog/assets/src/bootstrap/Makefile	2013-05-11 17:15:14.000000000 +0200
@@ -22,7 +22,7 @@
 	@./node_modules/.bin/recess --compile ${BOOTSTRAP_RESPONSIVE_LESS} > ${BOOTSTRAP_RESPONSIVE}
 	@echo "Compiling LESS with Recess...               ${CHECK} Done"
 	@node docs/build
-	@cp img/* docs/assets/img/
+	@cp font/* docs/assets/font/
 	@cp js/*.js docs/assets/js/
 	@cp js/tests/vendor/jquery.js docs/assets/js/
 	@echo "Compiling documentation...                  ${CHECK} Done"
@@ -62,7 +62,7 @@
 # recess & uglifyjs are required
 #
 
-bootstrap: bootstrap-img bootstrap-css bootstrap-js
+bootstrap: bootstrap-font bootstrap-css bootstrap-js
 
 
 #
@@ -92,14 +92,14 @@
 	./node_modules/.bin/recess --compress ${BOOTSTRAP_RESPONSIVE_LESS} > bootstrap/css/bootstrap-responsive.min.css
 
 #
-# IMAGES
+# FONTS
 #
 
-bootstrap-img: bootstrap/img/*
+bootstrap-font: bootstrap/font/*
 
-bootstrap/img/*: img/*
-	mkdir -p bootstrap/img
-	cp img/* bootstrap/img/
+bootstrap/font/*: font/*
+	mkdir -p bootstrap/font
+	cp font/* bootstrap/font/
 
 
 #
@@ -123,4 +123,4 @@
 	watchr -e "watch('less/.*\.less') { system 'make' }"
 
 
-.PHONY: docs watch gh-pages bootstrap-img bootstrap-css bootstrap-js
\ No newline at end of file
+.PHONY: docs watch gh-pages bootstrap-font bootstrap-css bootstrap-js
\ No newline at end of file
